RELEASE.md

```diff
@@ -16,6 +16,11 @@
 ## v0.1.10 - 04/13/2023
 - Added OC2 Realm.
 
+## v1.1.10 - 01/19/2023
+- Added OC3 Realm.
```

Comment:
may need more details on here. For which variable(s) we add the oc3 realm.

Reply:
Removed this line